### ISSUE TITLE
How can you get enough "aligned" images for pre-training?

### ISSUE THREAD

**Issue by GouZi2019** [alanqrwang/keymorph#20]
The accuracy and robustness of the selection of key points seems to be crucial, which depend on "Self-supervised Pretraining"

However, this step needs "a single subject and its set of **aligned** different-modality scans"

I wonder how can you get enough aligned images to train this module?

Looks like we need to do a groupwise registration first, right?

**Comment by GouZi2019** [comment]
Also, if you train on the images of only one subject, can you generalize to other subjects?

Thanks
